### PR TITLE
Translate pol.json and Default Settings SoD_SP.ini

### PR DESCRIPTION
**Pull request by pawel54321**
[+] Completly Translation for language Polish.
[+] Repaired polish characters.
[+] (Optional) Example Settings for language Polish.

### CODE REVIEW

File: SoD_SP_Public/Next Version/SoD_SP.ini
Comment (RoseKavalier) [inline SoD_SP_Public/Next Version/SoD_SP.ini:34]: The default ini options should not be modified, just the comments explaining what they do.

File: SoD_SP_Public/Next Version/SoD_SP.ini
Comment (RoseKavalier) [inline SoD_SP_Public/Next Version/SoD_SP.ini:40]: The default ini options should not be modified, just the comments explaining what they do.

File: SoD_SP_Public/Next Version/SoD_SP.ini
Comment (RoseKavalier) [inline SoD_SP_Public/Next Version/SoD_SP.ini:46]: The default ini options should not be modified, just the comments explaining what they do.

File: SoD_SP_Public/Next Version/SoD_SP.ini
Comment (RoseKavalier) [inline SoD_SP_Public/Next Version/SoD_SP.ini:52]: The default ini options should not be modified, just the comments explaining what they do.

File: SoD_SP_Public/Next Version/SoD_SP.ini
Comment (RoseKavalier) [inline SoD_SP_Public/Next Version/SoD_SP.ini:58]: The default ini options should not be modified, just the comments explaining what they do.

File: SoD_SP_Public/Next Version/SoD_SP.ini
Comment (RoseKavalier) [inline SoD_SP_Public/Next Version/SoD_SP.ini:64]: The default ini options should not be modified, just the comments explaining what they do.

File: SoD_SP_Public/Next Version/SoD_SP.ini
Comment (RoseKavalier) [inline SoD_SP_Public/Next Version/SoD_SP.ini:97]: The default ini options should not be modified, just the comments explaining what they do.

File: SoD_SP_Public/Next Version/SoD_SP.ini
Comment (RoseKavalier) [inline SoD_SP_Public/Next Version/SoD_SP.ini:109]: The default ini options should not be modified, just the comments explaining what they do.

File: SoD_SP_Public/Next Version/SoD_SP.ini
Comment (RoseKavalier) [inline SoD_SP_Public/Next Version/SoD_SP.ini:116]: The default ini options should not be modified, just the comments explaining what they do.

File: SoD_SP_Public/Next Version/SoD_SP.ini
Comment (RoseKavalier) [inline SoD_SP_Public/Next Version/SoD_SP.ini:184]: The default ini options should not be modified, just the comments explaining what they do.

File: SoD_SP_Public/Next Version/SoD_SP.ini
Comment (RoseKavalier) [inline SoD_SP_Public/Next Version/SoD_SP.ini:191]: The default ini options should not be modified, just the comments explaining what they do.

File: SoD_SP_Public/Next Version/SoD_SP.ini
Comment (RoseKavalier) [inline SoD_SP_Public/Next Version/SoD_SP.ini:198]: The default ini options should not be modified, just the comments explaining what they do.

File: SoD_SP_Public/Next Version/SoD_SP.ini
Comment (RoseKavalier) [inline SoD_SP_Public/Next Version/SoD_SP.ini:205]: The default ini options should not be modified, just the comments explaining what they do.

File: SoD_SP_Public/Next Version/SoD_SP.ini
Comment (RoseKavalier) [inline SoD_SP_Public/Next Version/SoD_SP.ini:217]: The default ini options should not be modified, just the comments explaining what they do.

File: SoD_SP_Public/Next Version/SoD_SP.ini
Comment (RoseKavalier) [inline SoD_SP_Public/Next Version/SoD_SP.ini:241]: The default ini options should not be modified, just the comments explaining what they do.

File: SoD_SP_Public/Next Version/SoD_SP.ini
Comment (RoseKavalier) [inline SoD_SP_Public/Next Version/SoD_SP.ini:253]: The default ini options should not be modified, just the comments explaining what they do.

File: SoD_SP_Public/Next Version/SoD_SP.ini
Comment (RoseKavalier) [inline SoD_SP_Public/Next Version/SoD_SP.ini:259]: The default ini options should not be modified, just the comments explaining what they do.

File: SoD_SP_Public/Next Version/SoD_SP.ini
Comment (RoseKavalier) [inline SoD_SP_Public/Next Version/SoD_SP.ini:309]: The default ini options should not be modified, just the comments explaining what they do.

File: SoD_SP_Public/Next Version/SoD_SP.ini
Comment (RoseKavalier) [inline SoD_SP_Public/Next Version/SoD_SP.ini:330]: The default ini options should not be modified, just the comments explaining what they do.

File: SoD_SP_Public/Next Version/SoD_SP.ini
Comment (RoseKavalier) [inline SoD_SP_Public/Next Version/SoD_SP.ini:360]: The default ini options should not be modified, just the comments explaining what they do.

File: SoD_SP_Public/Next Version/SoD_SP.ini
Comment (RoseKavalier) [inline SoD_SP_Public/Next Version/SoD_SP.ini:366]: The default ini options should not be modified, just the comments explaining what they do.

**Review by RoseKavalier — CHANGES_REQUESTED** [review]
Language looks very well, fantastic job - must have taken quite a while! Other translators have been doing this work over a few years so doing that in a few days is 👍 

As you can see from the copy-pasted comments, the default ini options should not be modified, that's the one thing missing before approval for merging.

Once again, good job!

**Comment by pawel54321** [comment]
I renamed.

Thank you very much for great project ;).

**Review by RoseKavalier — APPROVED** [review]
Looks good!

**Pull request merged**
